administrator/language/de-DE/de-DE.com_associations.ini

```diff
@@ -21,6 +21,7 @@ COM_ASSOCIATIONS_EDIT_ASSOCIATION="Verknüpfung bearbeiten"
 COM_ASSOCIATIONS_EDIT_HIDE_REFERENCE="Referenz verstecken"
 COM_ASSOCIATIONS_EDIT_SHOW_REFERENCE="Referenz anzeigen"
 COM_ASSOCIATIONS_ERROR_NO_ASSOC="Die Komponente „Sprachverknüpfung“ kann nicht genutzt werden, da die Website nicht mehrsprachig oder die Sprachverknüpfung im Plugin „<a href=\"%s\">System - Sprachfilter</a>“ nicht aktiv ist."
+COM_ASSOCIATIONS_ERROR_NO_TYPE="Der ausgewählte Eintragstype existiert nicht in dieser Erweiterung."
```

Comment:
Der ausgewählte Eintragstyp existiert nicht in dieser Erweiterung.

Reply:
fixed